shared/js/media/remote_controls.js

```diff
@@ -197,11 +180,6 @@ MediaRemoteControls.prototype._setupBluetooth = function(callback) {
       self._commandHandler(AVRCP.PLAY_PRESS);
     }
   }
-
-  function resetDefaultAdapter() {
-    self.defaultAdapter = null;
-    // Do we need to do anything else?
-  }
```

Comment:
I think it's better to add  callback to reset the adapter while Gecko::Bluetooth is not available for use.
* v1: `ondisabled` event(If you do so, please manual test for those user story(media playback, call screen via headset, pairing device) again. The major influence is using Bluetooth while Bluetooth just be turned off. These apps might do something from the useless adapter. But we already set default adapter to be null.)
* v2: `onattributechanged` of `BluetoothManager`(mozBluetooth) (But we know new adapter and null adapter is coming from the same `defaultAdapter` attribute. In other words, we no need to do any for v2.)